Dockerfile.dev

```diff
@@ -1,15 +1,16 @@
-FROM betagouv/transport:0.4.3
+FROM betagouv/transport:elixir-1.12.2-erlang-24.0.3-alpine-3.13.3
 
 RUN apk add inotify-tools
 RUN apk add postgresql-client>=11
+RUN apk add git
 
 RUN mkdir /app/
 RUN mkdir /app/_build
 RUN mkdir /app/deps/
 WORKDIR /app/
 
 # fetch a wait-for-it script to wait for postgres startup
-ADD https://raw.githubusercontent.com/vishnubob/wait-for-it/54d1f0bfeb6557adf8a3204455389d0901652242/wait-for-it.sh /opt/bin/
+ADD https://raw.githubusercontent.com/vishnubob/wait-for-it/81b1373f17855a4dc21156cfe1694c31d7d1792e/wait-for-it.sh /opt/bin/
```

Comment:
Le diff pour les intéressés:

https://github.com/vishnubob/wait-for-it/compare/54d1f0bfeb6557adf8a3204455389d0901652242..81b1373f17855a4dc21156cfe1694c31d7d1792e